### PR TITLE
Rename TraceOptions to TraceFlags to be w3c compatible

### PR DESCRIPTION
**Pull request by bogdandrutu**
See more details here https://www.w3.org/TR/trace-context/#traceparent-header-field-values

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Ping, I need few more approves

File: specification/api-tracing.md [hunk 1]
```diff
@@ -177,8 +177,8 @@ non-zero byte.
 `SpanId` A valid span identifier is an 8-byte array with at least one non-zero
 byte.
 
-`TraceOptions` contain details about the trace. Unlike Tracestate values,
-TraceOptions are present in all traces. Currently, the only TraceOption is a
+`TraceFlags` contain details about the trace. Unlike Tracestate values,
+TraceFlags are present in all traces. Currently, the only TraceOption is a
```
Comment (fbogsany) [inline specification/api-tracing.md]: s/TraceOption/TraceFlag/

File: specification/api-tracing.md [hunk 2]
```diff
@@ -177,8 +177,8 @@ non-zero byte.
...
 boolean `recorded`
```
Comment (fbogsany) [inline specification/api-tracing.md]: The trace-context spec calls this flag `sampled`, not `recorded`.

Comment (bogdandrutu) [reply specification/api-tracing.md]: Done

File: specification/api-tracing.md (hunk 1 above)
Comment (bogdandrutu) [reply specification/api-tracing.md]: Done

**Comment by bogdandrutu** [comment]
Approvers please approve this PR. I think this is a trivial (and good) change.

**Pull request merged**